charts/probate-back-office/Chart.yaml

```diff
@@ -2,14 +2,14 @@ apiVersion: v2
 description: A Helm chart for the HMCTS probate back office service
 name: probate-back-office
 home: https://github.com/hmcts/probate-back-office
-version: 1.0.24
+version: 1.0.25
 
 maintainers:
   - name: HMCTS Probate Team
     email: probate-action-group@HMCTS.NET
 dependencies:
   - name: java
-    version:  3.7.4
+    version:  4.0.1
```

Comment:
3 -> 4 is a breaking change, see
https://github.com/hmcts/chart-java/releases/tag/4.0.0

Reply:
thanks @nigeldunne, updated to latest 3.7.9 instead of 4